spec/factories/folders.rb

```diff
@@ -1,6 +1,7 @@
 FactoryGirl.define do
   factory :folder do
     sequence(:name) { |n| "Folder #{n}" }
+    is_private true
```

Comment:
I'm not sure if this line is necessary? The schema defaults to true, and the application will create any new item to be the privacy setting of the parent or change the privacy setting of the children when the parent is changed, but there's nothing on the model to do this. 

I'll investigate if we can use a before_action in order to lean out the controller and reduce any manual assignments for our tests.